elements/pfelement/src/pfelement.js

```diff
@@ -289,6 +289,13 @@ class PFElement extends HTMLElement {
     this._pfeClass = pfeClass;
     this.tag = pfeClass.tag;
     this._parseObserver = this._parseObserver.bind(this);
+    if (!this.id) this.id = this.randomId;
```

Comment:
```suggestion
    if (!this.id) this.id = this.randomId;
```
^ @kylebuch8 Do you see any issue with us assigning a randomId to all components if no ID was provided?